app/assets/javascripts/guides/new.js

```diff
@@ -48,6 +48,49 @@ openFarmApp.directive('formChecker', function(){
   };
 });
 
+openFarmApp.directive('clearOn', function() {
+   return function(scope, elem, attr) {
+      scope.$on('clearOn', function(e, name) {
+        if(name === attr.clearOn) {
+          elem[0].value = "";
```

Comment:
Hmm in my mind, one of these is a symbol and the other is a string, but maybe that's just me ¬_¬